test/mbedls_toolsbase.py

```diff
@@ -246,28 +247,34 @@ def test_fs_never(self):
             'mount_point': 'invalid_mount_point',
             'serial_port': 'invalid_serial_port'
         }
+        self.base.return_value = [device]
+        self.base.list_unmounted = True
         with patch("mbed_lstools.lstools_base.MbedLsToolsBase._update_device_from_fs") as _up_fs:
             filter = None
-            ret = self.base._fs_never(deepcopy(device), filter, False)
-            ret_with_details = self.base._fs_never(deepcopy(device), filter, True)
-            self.assertIsNotNone(ret)
-            self.assertIsNotNone(ret_with_details)
-            self.assertEqual(ret['target_id'], ret['target_id_usb_id'])
-            self.assertEqual(ret, ret_with_details)
+            ret = self.base.list_mbeds(FSInteraction.Never, filter, read_details_txt=False)
```

Comment:
Let's change the other tests `test_fs_after` and `test_fs_before` to go through `list_mbeds` as well so the cases are consistent.